overlord/snapstate/handlers.go

```diff
@@ -337,6 +337,15 @@ func (m *SnapManager) installOneBaseOrRequired(t *state.Task, snapName string, c
 	return ts, err
 }
 
+func isSeeded(st *state.State) (bool, error) {
```

Comment:
We don't have this function defined anywhere? I couldn't find it. Maybe someone could point me to it?

Reply:
we don't, we always just write the code. The issue is that the natural place to put the function would be devicestate but devicestate imports most of the rest of the managers so it gets messy